### ISSUE TITLE
Firmware Update not triggered

### ISSUE THREAD

**Issue by iamacarpet** [OSDeploy/OSDCloud#11]
Hello,

Thanks again for the great project, we are loving it! :).

We noticed the firmware update does not seem to be triggered automatically, even when the option is enabled.

I can see the firmware being downloaded from Microsoft Update & placed in C:\Drivers, where it can be triggered manually once logged in.

We expected it to be automatically applied as part of the driver installation on specialize.

Is this intentional? Or are we missing something?

We've tested on several `Dell Latitude` models, most notably the `Latitude 5520`.

Regards,
iamacarpet

**Comment by OSDeploy** [comment]
Yes that's out of my control.  Microsoft Update Catalog is typically 3-6 months behind on Firmware Updates from what Dell has, but if your Firmware is 2 years old, then OSDCloud may get it better than it was ...

**Comment by iamacarpet** [comment]
Understood, I might dig in and see if any of your existing code for pulling Dell drivers is re-usable to pull the latest firmware from them instead.
